english/marshmallow-attack/make-smores/start-to-cry/Start-to-cry.md

```diff
@@ -0,0 +1,3 @@
+It's been a BAD DAY!!!!!
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.